example/pom.xml

```diff
@@ -74,7 +74,17 @@ Also contains a sample Docker Compose file (see &quot;docker&quot; folder) for s
       <resource>
         <directory>${project.basedir}/docker</directory>
         <filtering>true</filtering>
-        <targetPath>${project.build.directory}/hono</targetPath>
+        <targetPath>${project.build.directory}/hono/example/docker</targetPath>
+      </resource>
+      <resource>
+        <directory>${project.basedir}/openshift</directory>
```

Comment:
I'm ok with these changes but I'm starting to think that this project isn't an example anymore ... even if it's really is because it's one of the possible Hono deployments. Moving from using "example" to "deploy" inside a path as suggested by @sophokles73 let me think that. Maybe we should rename the entire project ?